camus-shopify/script/upload_to_gcs.py

```diff
@@ -0,0 +1,199 @@
+#####
+# Upload directories imported by Camus to Google Cloud
+#####
+import argparse
+import os
+import subprocess
+import logging
+
+
+HDFS_PREFIX = "hdfs://hadoop-production"
+GCS_PREFIX = "gs://longboat"
+MIN_FILES_FOR_DIST = 8  # use regular copy for small number of files
+DIST_MAPPERS = 32  # allowed parallelism for distributed copy
+DIST_QUEUE = 'root.default'  # yarn queue for distributed copy
```

Comment:
let's try using `production.ignore` for now, that's what starscream uses for distcp, I'll see if the plan is getting all gcs distcps in the same queue